src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -121,6 +128,8 @@ export class SkyListViewGridComponent
 
   public currentSearchText: Observable<string>;
 
+  private _multiselectSelectedIds: string[] = [];
```

Comment:
Is this member tied to a `get` or `set`? We usually reserve the underscore for getter/setter values.

Reply:
got it. fixed.